lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * dummylayout.js
+ *
+ * this dumps a fake graph
+ *
+ * @author John O'Brien.
+ */
+
+var Node = require('../model/node.js');
+var Graph = require('../model/graph.js');
+var Port = require('../model/port.js');
+
+// Create a graph instance.
+var graph;
+
+function buildGraph(nodes_and_connections) {
+  var nodes = [];
+  //loop for building the nodes
+  for(var i =  0; i < 4; i++){
+  
+    //creates the ports
+    var portIn = new Port( ( i * 100 ) + 01, "input");
```

Comment:
Style: strings must use singlequotes. 
Tip: setting up jshint with your editor using the settings in the style guide will help find things.